contributors.js

```diff
@@ -18,5 +18,6 @@ const contributors = [
 	{ username: 'zeroual', message: 'Hellow hacktoberfest from Morocco!' },
   { username: 'FaustineG', message: 'Coucou le Discord :)' },
   { username: 'JollyJolli', message: 'Cuak, Cuak!' },
-  { username: 'JayShukla8', message: 'Autumn\'s here!' }
+  { username: 'JayShukla8', message: 'Autumn\'s here!' },
+  { username: 'rishi-harti768 ', message: 'Happy Hacktoberfest 24! wow!' }
```

Comment:
there is a space on your username 😅 
The username have to be exactly the same that your duck file